diff --git a/CMakeLists.txt b/CMakeLists.txt
index b3bf3a4..962b76d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,13 +27,7 @@ ament_auto_find_build_dependencies()
 
 # Determine the architecture
 if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
-  if(CMAKE_DEVICE STREQUAL "sbsa")
-    set(GXF_EXT_LIB_PATH "${CMAKE_CURRENT_SOURCE_DIR}/lib/gxf_aarch64_cuda_13_0")
-  elseif(CMAKE_DEVICE STREQUAL "arm64")
-    set(GXF_EXT_LIB_PATH "${CMAKE_CURRENT_SOURCE_DIR}/lib/gxf_jetpack70")
-  else()
-    message(FATAL_ERROR "Device is not supported.")
-  endif()
+  set(GXF_CORE_LIB_DIR gxf/core/lib/gxf_aarch64_cuda_13_0/)
 elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64")
   set(GXF_EXT_LIB_PATH "${CMAKE_CURRENT_SOURCE_DIR}/lib/gxf_x86_64_cuda_13_0")
 else()
@@ -48,7 +42,7 @@ install(FILES "${GXF_EXT_LIB_PATH}/lib${PROJECT_NAME}.so"
         DESTINATION "${CMAKE_INSTALL_PREFIX}/share/${PROJECT_NAME}/gxf/lib")
 set_property(TARGET ${PROJECT_NAME} PROPERTY
   INTERFACE_LINK_LIBRARIES
-    "$<INSTALL_PREFIX>/share/${PROJECT_NAME}/gxf/lib/lib${PROJECT_NAME}.so"
+    "$<INSTALL_INTERFACE/share/${PROJECT_NAME}/gxf/lib/lib${PROJECT_NAME}.so>"
 )
 
 # Install headers
@@ -69,8 +63,10 @@ ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)
 
 
 # Embed versioning information into installed files
-ament_index_get_resource(ISAAC_ROS_COMMON_CMAKE_PATH isaac_ros_common_cmake_path isaac_ros_common)
-include("${ISAAC_ROS_COMMON_CMAKE_PATH}/isaac_ros_common-version-info.cmake")
-generate_version_info(${PROJECT_NAME})
+if(NOT CMAKE_CROSSCOMPILING)
+  ament_index_get_resource(ISAAC_ROS_COMMON_CMAKE_PATH isaac_ros_common_cmake_path isaac_ros_common)
+  include("${ISAAC_ROS_COMMON_CMAKE_PATH}/isaac_ros_common-version-info.cmake")
+  generate_version_info(${PROJECT_NAME})
+endif()
 
-ament_auto_package(INSTALL_TO_SHARE)
\ No newline at end of file
+ament_auto_package(INSTALL_TO_SHARE)
